src/main/java/uk/gov/pay/ledger/report/entity/PerformanceReportEntity.java

```diff
@@ -1,10 +1,18 @@
 package uk.gov.pay.ledger.report.entity;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.math.BigDecimal;
 
 public class PerformanceReportEntity {
+
+    @JsonProperty("total_volume")
```

Comment:
[Minor]
```java
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```
could be used at the class level instead of defining property names explicitly.